lib/puma/request.rb

```diff
@@ -12,6 +12,13 @@ module Puma
   #
   module Request
 
+    LEN_BUFFER   = 1_024 * 256
+    STRM_LEN_MAX = 1_024 * 1_024 * 4
+
+    SKT_WRITE_ERR_MSG = "Socket timeout writing data"
```

Comment:
I would like if we used "socket" in the code over "skt"

Reply:
Good idea.  See above comment.

Reply:
@dentarg - fixed